datasets/DLUHC-Domestic-Energy-Performance-Certificates-for-existing-dwellings-by-energy-efficiency-rating-2022/main.py

```diff
@@ -0,0 +1,128 @@
+import pandas as pd
+import json
+from gssutils import *
+
+info = json.load(open('info.json'))
+title_id = info['id']
+
+metadata = Scraper(seed="info.json")
+metadata.dataset.title = "Domestic Energy Performance Certificates for existing dwellings by energy efficiency rating 2022"
```

Comment:
Please include 
metadata.dataset.comment = 'Data from certificates lodged on the Energy Performance of existing Buildings Register by average energy efficiency rating.'